docs/contributing.rst

```diff
@@ -1,10 +1,535 @@
-Contributing to SPRAS
-=====================
+SPRAS contribution guide
+========================
 
-We welcome contributions to SPRAS! If you would like to contribute, please follow the guidelines below.
+The SPRAS maintainers welcome external contributions. Code contributions
+will be licensed using the project’s `MIT license <LICENSE>`__.
 
-Adding New Network Algorithms
------------------------------
+Contributing a new pathway reconstruction guide
+-----------------------------------------------
 
-Helping with Other Code
```

Comment:
Are all the manual changes isolated to this part of the guide?

Reply:
There were no manual changes - this is all from CONTRIBUTING.md.

Reply:
I see, `docs/contributing.rst` was an existing stub and the diff is relative to that.